security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc

```diff
@@ -18,6 +18,6 @@ include::modules/compliance-supported-profiles.adoc[leveloffset=+1]
 [role="_additional-resources"]
 == Additional resources
 
-* xref:../../security/compliance_operator/compliance-operator-understanding.adoc#compliance_profiles_understanding-compliance[Compliance Operator profiles]
+* xref:../../../security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc#compliance_profiles_understanding-compliance[Compliance Operator profiles]
 
-* xref:../../security/compliance_operator/compliance-operator-understanding.adoc#compliance_profile_typesunderstanding-compliance[Compliance Operator profile types]
\ No newline at end of file
+* xref:../../../security/compliance_operator/co-concepts/compliance-operator-understanding.adoc#compliance_profile_typesunderstanding-compliance[Compliance Operator profile types]
```

Comment:
This link is also in `compliance-operator-supported-profiles.adoc` assembly; I think it is formatted incorrectly: #compliance_profile_typesunderstanding-compliance

> xref:../../security/compliance_operator/compliance-operator-understanding.adoc#compliance_profile_typesunderstanding-compliance[Compliance Operator profile types]